code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -65,6 +72,13 @@
 	if(energy != oldenergy)
 		use_power(2500)
 
+/obj/machinery/chem_dispenser/emag_act(mob/user)
+	if(emagged || !emagged_reagents)
+		user << "<span class='warning'>\The [src] has no functional safeties to emag.</span>"
+		return
+	user << "<span class='notice'>You short out \the [src]'s safeties.</span>"
+	dispensable_reagents |= emagged_reagents//add the emagged reagents to the dispensable ones
```

Comment:
ohhh shit i'm an idiot, i figured emag_act set emagged automatically (despite me never even calling parent)

the check for emagged_reagents is to ensure the machine actually has an emagged_reagents list but I could remove it if it isn't needed